src/renderer/js/page/subscriptions/view.jsx

```diff
@@ -0,0 +1,106 @@
+// @flow
+import React from "react";
+import SubHeader from "component/subHeader";
+import { BusyMessage } from "component/common.js";
+import { FeaturedCategory } from "page/discover/view";
+import type { Subscription } from "redux/reducers/subscriptions";
+
+type SavedSubscriptions = Array<Subscription>
+
+type Props = {
+  doFetchClaimsByChannel: (string, number) => any,
+  savedSubscriptions: SavedSubscriptions,
+  // TODO build out claim types
+  subscriptions: Array<any>,
+  setHasFetchedSubscriptions: () => void,
+  hasFetchedSubscriptions: boolean
+};
+
+export default class extends React.PureComponent<Props> {
+  // setHasFetchedSubscriptions is a terrible hack
```

Comment:
I'm okay shipping it like this, but this could be an example where the empty vs. undefined is a useful distinction. If the subscription array starts undefined, then we can differentiate between never fetched (undefined) and fetched but empty (empty array).